docs/assets/logo.svg

```diff
@@ -1,14 +0,0 @@
-<svg width="95" height="128" viewBox="0 0 95 128" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Odmazání toho loga rozbije logo v readme toho repa. Co s tím? Asi bych tam to staré logo nechal taky.

Reply:
@jmarek41 fixed